src/applications/apps.js

```diff
@@ -59,7 +59,12 @@ export function registerApplication(appName, applicationOrLoadingFn, activityFn,
     activeWhen: activityFn,
     status: NOT_LOADED,
     parcels: {},
-    devtools: {},
+    devtools: {
+      overlays: {
+        options: {},
+        selectors: [],
+      }
+    },
```

Comment:
defaulting the devtools object to the structure I need to ensure it's always defined